app/javascript/src/PaymentGateways/index.js

```diff
@@ -0,0 +1,2 @@
+export { StripeFormWrapper } from 'PaymentGateways/components/StripeFormWrapper'
+export { CardForm } from 'PaymentGateways/components/CardForm'
```

Comment:
IMO it should be called `StripeCardForm`. Like this:
```suggestion
export { CardForm } from 'PaymentGateways/components/StripeCardForm'
```